mungegithub/mungers/approvers/README.md

```diff
@@ -0,0 +1,218 @@
+# Reviewers and Approvers
+
+## Questions this Doc Seeks To Answer
+
+1. What do we need to get a PR merged into kubernetes/kubernetes?
+2. What are reviewers, approvers, and the OWNERS files?
+3. How does the reviewer selection mechanism work? approver selection mechanism work?
+4. How does an approver know which PR s/he has to approve?
+
+## Overview
+
+Every GitHub directory which is a unit of independent code contains a file named "OWNERS". The file lists reviewers and approvers for the directory. Approvers (or previously called assignees) are owners of the codes. 
+
+Approvers:
+* have contributed substantially to the project
+* can provide a final approval (`/approve`) indicating whether a change to a directory or subdirectory should be accepted
+* Approval is done on a per directory basis and subdirectories inherit their parents directory's approvers
+
+Reviewers:
+* generally a larger set of current and past contributors
+* They are responsible for a more thorough code review, discussing the implementation details and style
+* Provide an `/lgtm` when they are satisfied with the Pull Request 
+
+An example of the OWNERS file is listed below:
+
+reviewers:
+* jack
+* ken
+* lina
+
+approvers:
+* jack
+* ken
+* lina
+
```

Comment:
Add this text: the items in the OWNERs file can be github usernames, or they can be aliases defined in the OWNERS_ALIASES file, which is a mechanism for defining groups.  Github Team names are not supported.    We do not use GH Teams because there is no audit log for changes to GH Teams.  This way we have an audit log.